### ISSUE TITLE
SpotlightAPI Missing 'batchrsp/items' error (probably to api changes)

### ISSUE THREAD

**Issue by perdigano** [ORelio/Spotlight-Downloader#21]
It seems that API endpoint has somehow changed.
A manual call to url "**https://arc.msn.com/v3/Delivery/Cache?pid=209567&fmt=json**" (or with any with additional parameters) obtain this json response:

```
{
 "batchrsp":
 {
   "ver":"1.0",
   "errors":
   [
    {
      "code":2000,
      "msg":"No ad available. No additional information"
     }
   ],
   "refreshtime":"2021-01-13T13:16:20"
  }
}
```
In case I discover more information I'll update this message.

Best regards and thanks for this project!

**Comment by KDVGitHub** [comment]
+1 Same error

**Comment by ORelio** [comment]
I think I've found the updated URL. Releasing a new build with updated URL.

**Comment by ORelio** [comment]
Fixed as per release v1.4.4. Thanks for reporting!
